pages/api/dial_out.ts

```diff
@@ -0,0 +1,53 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+
+import { notify } from 'lib/bugsnag';
+
+type Data = {};
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
+  if (req.method === 'POST') {
+    if (
+      req.body &&
+      req.body.video_room_id &&
+      req.body.video_room_context &&
+      req.body.to
+    ) {
+      try {
+        const response = await fetch(
+          `${process.env.TELNYX_CALL_CONTROL_API_HOST}/dial_out`,
+          {
+            method: 'POST',
+            body: JSON.stringify(req.body),
+            headers: {
+              'Content-Type': 'application/json',
+            },
+          }
+        );
+
+        if (response.ok) {
+          res.status(200).json(response);
```

Comment:
You must get the json from the response before you pass it down.

```ts
        if (response.ok) {
          const { data } = await response.json();
          res.status(200).json(data);
        } else {
 ```

Reply:
Got it!

Note that the server does not sends `data` with the response; I'll pass down the whole json instead, which in this case is `{ result: 'OK' }`

Resolved in: [c6d3af3](https://github.com/team-telnyx/telnyx-meet/pull/70/commits/c6d3af3a222ceee106466010bc0a547276caf0f2)

Reply:
I've added a fix to the API request in [a439db1](https://github.com/team-telnyx/telnyx-meet/pull/70/commits/a439db1fb5e887ec67af740f17c42ab573db936e), as the context inside the body must be stringify(ed) too.